packages/dx-react-grid-material-ui/src/templates/group-panel-item.test.jsx

```diff
@@ -83,4 +86,89 @@ describe('GroupPanelItem', () => {
     expect(tree.find('Chip').props())
       .toHaveProperty('onRequestDelete');
   });
+
+  it('should not call change sorting func if not allow sorting', () => {
+    const changeSortingDirection = jest.fn();
+    const tree = mount((
+      <GroupPanelItem
+        changeSortingDirection={changeSortingDirection}
+        column={{
+          name: 'test',
+        }}
+      />
+    ));
+    const ChipElem = tree.find(Chip);
+
+    ChipElem.simulate('keydown', { keyCode: ENTER_KEY_CODE });
+    expect(changeSortingDirection)
+      .not.toHaveBeenCalled();
+  });
+
+  it('should handle the "Enter" and "Space" keys down and "Mouse click" for sorting change', () => {
+    const changeSortingDirection = jest.fn();
+    const tree = mount((
+      <GroupPanelItem
+        changeSortingDirection={changeSortingDirection}
+        column={{
+          name: 'test',
+        }}
+        allowSorting
+      />
+    ));
+    const ChipElem = tree.find(Chip);
+
+    ChipElem.simulate('keydown', { keyCode: ENTER_KEY_CODE });
+    expect(changeSortingDirection)
+      .toHaveBeenCalled();
+
+    changeSortingDirection.mockClear();
+    ChipElem.simulate('keydown', { keyCode: SPACE_KEY_CODE });
+    expect(changeSortingDirection)
+      .toHaveBeenCalled();
+
+    changeSortingDirection.mockClear();
+    ChipElem.simulate('click');
+    expect(changeSortingDirection)
+      .toHaveBeenCalled();
+
+    changeSortingDirection.mockClear();
+    ChipElem.simulate('keydown', { keyCode: 51 });
+    expect(changeSortingDirection)
+      .not.toHaveBeenCalled();
+  });
+
+  it('should handle the "Ctrl" key with sorting', () => {
```

Comment:
'should cancel sorting on sorting direction change when the "Ctrl" key is pressed'